fabric/src/main/java/me/lunaluna/fabric/elytrarecast/mixin/PlayerMixin.java

```diff
@@ -29,16 +29,16 @@ private ClientPlayerEntity player() {
     private boolean awaitingElytra = false;
 
     @SuppressWarnings("ConstantConditions")
-    @Inject(method = "tickFallFlying", at = @At("TAIL"))
+    @Inject(method = "tickMovement", at = @At("TAIL"))
```

Comment:
The is a `tickGliding` that replaced `tickFallFlying` but it doesn't work at all as you never get a gliding tick that would have `isGliding` as false. So I opted for `tickMovement` which seems to work but is obviously far less efficient.

Reply:
This is fine, and was the way I was using previously